cmd/earthly/main.go

```diff
@@ -1176,6 +1176,82 @@ Set up a whole custom git repository for a server called example.com, using a si
 				},
 			},
 		},
+		{
+			Name: "preview",
+			Subcommands: []*cli.Command{
+				{
+					Name:        "project",
+					Description: "Create and manage Earthly projects",
+					UsageText:   "earthly project (ls|rm|create|member)",
+					Flags: []cli.Flag{
+						&cli.StringFlag{
+							Name:     "org",
+							EnvVars:  []string{"EARTHLY_ORG"},
+							Usage:    "The name of the organization to which the project belongs. Required when user is a member of multiple.",
+							Required: false,
+						},
+					},
+					Subcommands: []*cli.Command{
+						{
+							Name:        "ls",
+							Usage:       "List all projects that belong to the specified organization",
+							Description: "List all projects that belong to the specified organization",
+							UsageText:   "earthly project [--org <organization-name>] ls",
+							Action:      app.actionProjectList,
+						},
+						{
+							Name:        "rm",
+							Usage:       "Remove an existing project from the organization",
+							Description: "Remove an existing project from the organization",
+							UsageText:   "earthly project [--org <organization-name>] rm <project-name>",
+							Action:      app.actionProjectRemove,
+						},
+						{
+							Name:        "create",
+							Usage:       "Create a new project in the specified organization",
+							Description: "Create a new project in the specified organization",
+							UsageText:   "earthly project [--org <organization-name>] create <project-name>",
+							Action:      app.actionProjectCreate,
+						},
+						{
+							Name:        "member",
```

Comment:
For some reason, the help text for `member` doesn't show up...

<img width="966" alt="Screen Shot 2022-07-06 at 2 29 48 PM" src="https://user-images.githubusercontent.com/446771/177646864-4904a06e-7ff5-4d44-aea6-672e0b0df1c8.png">
<img width="966" alt="Screen Shot 2022-07-06 at 2 32 04 PM" src="https://user-images.githubusercontent.com/446771/177647172-1efc020a-da38-4ea7-9381-7c05877a63d2.png">

Reply:
Fixed. A description field was missing.